src/serialized-analysis.ts

```diff
@@ -0,0 +1,425 @@
+export interface SerializedAnalysis { packages: Package[]; }
+
+export interface Package {
```

Comment:
`elements.json`, I like it.

`version` was requested from the catalog folks, which prompted the concept of a package.

I'm ok with not duplicating info from bower.json and package.json and dropping those fields, I kinda threw them in on a lark. Do we ever want to represent multiple packages' worth of elements in one metadata file though? I suspect that we do for the catalog use case, where it would be nice to have one file that contained all of the metadata. For an `elements.json` it would make sense to serialize just one package, sure.